library/spdm_responder_lib/libspdm_rsp_vendor_response.c

```diff
@@ -11,15 +11,6 @@
 /* expected number of bytes for VENDOR MESSAGE HEADERS */
 #define SPDM_VENDOR_DEFINED_FIXED_HEADER_LEN 7
```

Comment:
Pre-existing issue, but I do not see a reason for this `#define` to exist. I will file a separate issue for it.